### ISSUE TITLE
Simulation result files and input files 

### ISSUE THREAD

**Issue by chraibi** [JuPedSim/jpscore#319]
In Gitlab by @gjaeger on Jul 4, 2019, 08:06 [[origin](https://gitlab.version.fz-juelich.de/jupedsim/jpscore/issues/332)]

**jpscore**: 

```
JuPedSim - JPScore

Current date   : Jul  2 2019 19:07:41
Version        : 0.8.4
Commit hash    : v0.8.4-68-ga33375a
Commit date    : Mon Jul 1 15:21:05 2019
Branch         : develop
----
```

**inifile** in `/data`:

```xml
<header>
  <seed>4313</seed>
  <max_sim_time>80</max_sim_time>
  <num_threads>48</num_threads>
  <show_statistics>true</show_statistics>
  <logfile>log_RiMEA_Test_04_7-0_om_2</logfile>
  <!--<progressbar/>-->
  <trajectories format="plain" fps="2">
    <file location="traj_RiMEA_Test_04_7-0_om_2.txt" />
  </trajectories>

  <geometry>geo_RiMEA_Test_04.xml</geometry>
</header>

...
```

**batch_script** in `$HOME`:

```batch
#!/bin/bash -x
#SBATCH -J R70
#SBATCH --account=ias-7
#SBATCH --nodes=1
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=48
#SBATCH --output=data/out_%j.txt
#SBATCH --error=data/err_%j.txt
#SBATCH --time=6:00:00
#SBATCH --mail-user=g.jaeger@fz-juelich.de
#SBATCH --mail-type=ALL

export OMP_NUM_THREADS=${SLURM_CPUS_PER_TASK}

srun ./jpscore/bin/jpscore data/ini_RiMEA_Test_04_7-0_om_2.xml
```

The logfile is stored in the root directory (`$HOME`) as `datalog_RiMEA_Test_04_7-0_om_2.txt`. Also the statistic files are stored there.

Both files should be stored in the directory `data` (like the trajectory files).

**Note**:    
The function `<progressbar/>` creates a very large file (5-15 GB) and should be disabled.

**Comment by anna-braun** [comment]
I started a simulation with Jureca today and I have a similar problem. The Trajectories have been written to the directory "Simulationen/test-01", but the statistic files and goal.vtk have been written to the root directory:
![Bildschirmfoto 2019-08-02 um 13 31 16](https://user-images.githubusercontent.com/38530070/62368121-f7601080-b52b-11e9-8fe9-61e4e5c10d1d.png)

@chraibi @schroedtert Is there any possibility to set a file location to "show_statistics"? If not, we should really add this function. For the KapaKrit simulations we will have many statistic files (100+/simulation), which will lead to total chaos, if all the files will be be written into the same/root directory.

**Comment by chraibi** [comment]
During the process of fixing  #448, these files (log, and flow*), I would propose to save these files in the same directory as the trajectory file. 

![Screenshot 2019-08-04 23 20 40](https://user-images.githubusercontent.com/5772973/62425694-8a58a080-b70e-11e9-8339-66ca2a77254d.png)

**Comment by anna-braun** [comment]
@zeroset 
1. As discussed today the structure for the output could look like this:
![Datenstruktur](https://user-images.githubusercontent.com/38530070/72244116-d973fd00-35ed-11ea-8261-8fae56cb2df2.png)
With an new output folder for every simulation, which contains all output files like trajectory files and flow statistics and a copy of all input files.     
2. Also important is the paths in the trajectory file. At the moment these are absolute paths:
![Pfade_ist](https://user-images.githubusercontent.com/38530070/72244489-ac741a00-35ee-11ea-83ba-af49b500682d.png)
But it should better be relative paths to have more flexibility when changing a folders name or sharing the data.
![Pfade_soll](https://user-images.githubusercontent.com/38530070/72244543-cb72ac00-35ee-11ea-95a0-e09d1a74cdef.png)

**Comment by schroedtert** [comment]
Can this be closed? See #649
